src/workspace-data/Data.js

```diff
@@ -712,6 +713,21 @@ export const WorkspaceData = _.flow(
       [signal]
     );
 
+    const loadWdsCapabilities = useCallback(
```

Comment:
Should `loadCapabilities` be a method of the WDS Provider? That could keep the capabilities state in the provider instead of duplicated it in the provider and component.

Reply:
I like this idea but I think knowing how to do this (and navigate the ways this changes the async code) exceeds my skills...can you show me how?

Reply:
I think it's not far removed from what's here now.

Something like a method on WDS Provider:
```ts
loadCapabilities: async (signal?: AbortSignal): Promise<void> => {
  const capabilitiesResult = await Ajax(signal).WorkspaceData.getCapabilities(this.proxyUrl);
  // update this.features based on capabilitiesResult
}
```

Then here, call `await wdsDataTableProvider.loadCapabilities();` instead of `await Ajax(signal).WorkspaceData.getCapabilities(url);`. Then Data doesn't need a `wdsCapabilities` state variable.

Reply:
Thanks, I'll give this a shot.  I was interpreting your suggestion to mean that the `getCapabilities` call needed to be handled somehow in the instantation of `wdsDataTableProvider` rather than beforehand like it's doing now.

Reply:
I discussed this in VC with @nawatts and we arrived on the conclusion that without some structural changes this is probably the most sensible spot to load the capabilities data since it only needs to be loaded once for a WDS instance, rather than once per data table.

A bit down the road, if we are able to split apart some of the mixed responsibilites of the `DataTableProvider` interface, this logic could move to become a concern _only of_ the parts of the interface concerned with listing data tables across all of a workspace.